wallets/blink/server.js

```diff
@@ -0,0 +1,61 @@
+import { withTimeout } from '@/lib/time'
+import { getScopes, SCOPE_READ, SCOPE_RECEIVE, SCOPE_WRITE, getWallet, request } from 'wallets/blink/common'
+import { msatsToSats } from '@/lib/format'
+export * from 'wallets/blink'
+
+export async function testCreateInvoice ({ apiKeyRecv, currencyRecv }) {
+  const scopes = await getScopes(apiKeyRecv)
+  if (!scopes.includes(SCOPE_READ)) {
+    throw new Error('missing READ scope')
+  }
```

Comment:
Interesting. I thought `RECEIVE` scope might be enough but I tested it and the scopes query returned `null`. So I guess `READ` is indeed needed to check that we weren't given `WRITE` permissons. Funny.

Reply:
i think it is also used to fetch the wallet id